scripts/xtensa-build-zephyr.py

```diff
@@ -641,7 +643,7 @@ def rimage_west_configuration(platform_dict, dest_dir):
 
 	platform_wconfig.set("rimage.extra-args", shlex.join(extra_args))
 
-	return platform_west_config_path
+	return platform_west_config_path, platform_wconfig.get("rimage.path")
```

Comment:
Good idea but you didn't change the corresponding pydoc.

Also, if you want to go down that route then return not just rimage.path but the whole `west.ConfigFile` object. This not just more in line with the function name but it also gives you access to `rimage.extra-args` and what not. Then you pass that west.ConfigFile object to `install_lib(..., platform_wconfig)` which has now access not just to `platform_wconfig.get("rimage.path")` but also to other rimage parameters.

Like this:

```diff
--- a/scripts/xtensa-build-zephyr.py
+++ b/scripts/xtensa-build-zephyr.py
@@ -594,7 +594,7 @@ RIMAGE_SOURCE_DIR = west_top / "sof" / "tools" / "rimage"
 def rimage_west_configuration(platform_dict, dest_dir):
 	"""Configure rimage in a new file `dest_dir/westconfig.ini`, starting
 	from the workspace .west/config.
-	Returns the pathlib.Path to the new file.
+	Returns tuple (west ConfigFile, pathlib.Path to that new file)
 	"""
 
 	saved_local_var = os.environ.get('WEST_CONFIG_LOCAL')
@@ -634,7 +634,7 @@ def rimage_west_configuration(platform_dict, dest_dir):
 
 	platform_wconfig.set("rimage.extra-args", shlex.join(extra_args))
 
-	return platform_west_config_path
+	return platform_wconfig, platform_west_config_path
```